setup.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+
+# This file will be the host environment setup file for all operating systems other than base Windows.
+
+if [[ "$OSTYPE" == "linux-gnu"* ]]; then
+  # Check if root or sudo
+  root=true
+  if [ "$EUID" -ne 0 ]; then
+    root=false
+  fi
+
+  distro="$(python -mplatform)"
+  if "$distro" | grep -qi "Ubuntu"; then
+    echo "Ubuntu detected."
+    echo "Installing Python TK if not found on the system."
+    if [ ! $(dpkg-query -W -f='${Status}' python3-tk 2>/dev/null | grep -c "ok installed") -eq 0 ]; then
+      if [ root = true ]; then
+        apt-get install python3-tk
+      else
+        echo "This script needs to be run as root or via sudo to install packages."
+        exit 1
+      fi
+    else
+      echo "Python TK found! Skipping install!"
+    fi
+  elif "$distro" | grep -Eqi "Fedora|CentOS|Redhat"; then
+    if ! rpm -qa | grep -qi python3-tkinter; then
+      if [ root = true ]; then
+        dnf install python3-tkinter
+      else
+        echo "This script needs to be run as root or via sudo to install packages."
+        exit 1
+      fi
```

Comment:
Can we add ArchLinux/Manjaro support? I believe the command to install tkinter is `pacman -S tk`

Reply:
Latest commit should fix this for Manjaro specifically, but also any other "family" we add. For instance, I added the Ubuntu family as well which should cover Mint Linux, Pop OS, etc.